rfcs/0000-rfc-process.md

```diff
@@ -0,0 +1,132 @@
+- Feature Name: rfc-process
+- Start Date: 2017-02-12
+- RFC PR:
+- Related Issue:
+
+# Summary
+[summary]: #summary
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for new features to enter the nix language, packages and
+OS, so that all stakeholders can be confident about the direction the
+ecosystem is evolving in.
+
+# Motivation
+[motivation]: #motivation
+
+There are a number of GitHub issues, PRs and mailing-list discussions where
```

Comment:
Are you sure you want to encode a specific hosting tool inside these documents? Why not say 'issues from the tracker'?